example_project/mappers/Baseline.rb

```diff
@@ -818,30 +596,27 @@ def find_building_for_uo_id(uo_buildstock_mapping_csv_path, feature)
               @@logger.error("\n uo_buildstock_mapping_csv_path was not assigned by the user")
             end
             
-            #############################################################
             ############### run uo-resstock workflow#####################
-
             # Run workflows if  UO-ResStock connection is established
             if uo_resstock_connection
 
               if !uo_buildstock_mapping_csv_path.nil?
                 ### If uo_buildstock_mapping_csv_path is provided
                 @@logger.info("Processing with UO-BuildStock mapping CSV path.")
-                resstock_building_id = find_building_for_uo_id(uo_buildstock_mapping_csv_path, feature)
-                puts "restock_building_id = #{resstock_building_id }"
+                resstock_building_id = find_building_for_uo_id(buildstock_csv_path, uo_buildstock_mapping_csv_path, feature)
+                puts "restock_building_id = #{resstock_building_id}"
                 ## run residential_samples
                 ## TODO: this should be developed to run with as much charachteristics are provided in the buildstock csv for the uo feature
                 require File.join(File.dirname(__FILE__), 'residential/samples/util')
```

Comment:
@joseph-robertson: need to move require before **find_building_for_uo_id** as this method moved to **'residential/samples/util'** too

Reply:
Good catch. I hadn't really been using/testing this functionality up to this point yet.